### PR TITLE
RDoc-1806-static-JS-counter-index

### PR DESCRIPTION
**Pull request by morhilai**
Documented AbstractJavaScriptCountersIndexCreationTask

### CODE REVIEW

File: Documentation/5.1/Raven.Documentation.Pages/document-extensions/counters/indexing.dotnet.markdown [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+{NOTE: }
+
+* To index counters, create a [static index](../indexes/creating-and-deploying#static-indexes) 
+that inherits from `AbstractCountersIndexCreationTask`.  
+
+* Auto-indexes for counters are not available at this time.  
+
+* In this page:  
+  * [Syntax](../../document-extensions/counters/indexing#syntax)  
+  * [Querying the Index](../../document-extensions/counters/indexing#querying-the-index)
```
Comment (ppekrol) [inline Documentation/5.1/Raven.Documentation.Pages/document-extensions/counters/indexing.dotnet.markdown:13]: Don't we miss here few items from this page?

Comment (morhilai) [reply Documentation/5.1/Raven.Documentation.Pages/document-extensions/counters/indexing.dotnet.markdown:15]: Fixed 👍

**Comment by ppekrol** [comment]
LGTM. @morhilai can you rebase?

**Comment by morhilai** [comment]
> LGTM. @morhilai can you rebase?

Done

**Pull request merged**